CONTRIBUTING.md

```diff
@@ -28,5 +28,25 @@ We use GitHub issues to track public bugs. Please ensure your description is cle
 * "Attractive"
 * Do not use the optional parameters of `setTimeout` and `setInterval`
 
+## Testing
+
+To run tests in the browser, run
+
+```
+npm run test:server
+```
+
+Open http://localhost:8081/test. Anytime you change test files, the tests will automatically be rerun, no refresh
+required! You can even click a specific section to only run those tests.
+
+To run tests in the CLI, run
+
+```
+npm run test
+```
+
+This will fail on some errors that npm run test:server won't, so it's recommended that you run this command after you've
+finished developing on tests with npm run test:server, if you chose that method.
```

Comment:
I've researched all I can, but still don't have a definite answer. My latest theory is that it's actually the difference between running in the browser and running in node.js. One observation is that there are new and different errors depending on node version, as can be seen in the travis results for my branch: https://travis-ci.org/schrodinger/fixed-data-table-2/builds/238907717?utm_source=github_status&utm_medium=notification.

But I can't be sure. Sorry!